app/javascript/packs/onboardingRedirectCheck.jsx

```diff
@@ -61,18 +60,26 @@ document.ready.then(
     }),
 );
 
+function shouldRedirectToOnboarding() {
+  // If the value is null in localStorage, that means we should redirect.
+  const shouldRedirect =
```

Comment:
This is using [null coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator).